packages/static-website/src/cloudfront-web-acl.ts

```diff
@@ -93,53 +100,201 @@ export class CloudfrontWebAcl extends Construct {
   constructor(scope: Construct, id: string, props?: CloudFrontWebAclProps) {
     super(scope, id);
 
+    const stack = Stack.of(this);
+    const aclName = `${stack.stackName}-${id}`; // Unique per stack
+    const onEventHandler = this.createOnEventHandler(stack, aclName);
+    const customResource = this.createAclCustomResource(
+      stack,
+      aclName,
+      onEventHandler,
+      props
+    );
+
+    this.webAclId = customResource.getAttString("WebAclId");
+    this.webAclArn = customResource.getAttString("WebAclArn");
+  }
+
+  /**
+   * Creates an event handler for managing an ACL in us-east-1.
+   *
+   * @param stack containing Stack instance.
+   * @param aclName name of the ACL to manage.
+   * @private
+   */
+  private createOnEventHandler(stack: Stack, aclName: string): Function {
+    const onEventHandlerName = `${stack.stackName}-OnEventHandler`;
+    const onEventHandlerRole = new Role(this, "OnEventHandlerRole", {
+      assumedBy: new ServicePrincipal("lambda.amazonaws.com"),
+      inlinePolicies: {
+        logs: new PolicyDocument({
+          statements: [
+            new PolicyStatement({
+              effect: Effect.ALLOW,
+              actions: [
+                "logs:CreateLogGroup",
+                "logs:CreateLogStream",
+                "logs:PutLogEvents",
+              ],
+              resources: [
+                `arn:aws:logs:${stack.region}:${stack.account}:log-group:/aws/lambda/${onEventHandlerName}`,
+                `arn:aws:logs:${stack.region}:${stack.account}:log-group:/aws/lambda/${onEventHandlerName}:*`,
```

Comment:
Nit: Could use `stack.formatArn` here and throughout?

Reply:
Will make this change in a follow-up PR.